docs/Project-Configuration/Github-Integrations.md

```diff
@@ -174,6 +173,66 @@ The check run will include a `view more details` link which will link back to th
 - Check runs only apply to PR patches
 - There is a limit on how many check runs can be added per version. This limit is set by evergreen based on demand and available resources and is subject to change.
 
+## Dynamic Github Access Tokens
+
+Evergreen offers the github.generate_token project command (documentation coming soon) that dynamically generates a short lived token from a user-provided GitHub app. Before you can use the command you will need to set up your github app in your project settings. This feature is not available as a [repo-level setting](/Project-Configuration/Repo-Level-Settings), it will need to be set up separately for each project where the command will be used.
+
+If you don't already have a github app, reach out to the IT team for help obtaining one.
+
+#### Saving An App Id and Key
+
+To get started, save your app ID and app key in your project settings in the GitHub App Settings tab. ![save-app.png](../images/github_app_id_and_key.png)
+
+Once an app id and key are saved, the app id will remain visible, but the app key will be redacted. If you need to make any changes to it, you can delete it and save it again.
+
+#### Restricting Generated Tokens
+
+When tokens are generated by the github.generate_token project command, they will have the full permissions of the github app.
+
+> While an owner and repository is required when generating a token from a github app (the project command will default to the owner and repository of the project if not specified), you cannot rely on the token being scoped down to that repository, as it may have the power to access other repositories in the org as well.
+
+Tokens generated by the github.generate_token can optionally be restricted directly in the github.generate_token command (link coming soon) and in project settings.
+
+To restrict it in project settings, start by creating a GitHub permission group in the GitHub Permission Groups tab. For a list of available permission types and levels, please take a look at `properties of permissions` in [the github documentation](https://docs.github.com/en/rest/apps/apps?apiVersion=2022-11-28#create-an-installation-access-token-for-an-app).
```

Comment:
I think this paragraph addresses two things: 1. project settings 2. github permission settings.

I think the 1st should go with the first sentence with the command restriction reference. And the 2nd should have its own paragraph (maybe even a ##### if you're feeling silly)

Reply:
Thanks! Moved the second part under the image